core/message_pool_test.go

```diff
@@ -65,31 +65,33 @@ func TestMessagePoolValidate(t *testing.T) {
 		require := require.New(t)
 		assert := assert.New(t)
 
+		// pull the default size from the default config value
+		maxMessagePoolSize := config.NewDefaultConfig().Mpool.MaxPoolSize
 		ctx := context.Background()
-		pool := NewMessagePool(th.NewTestMessagePoolAPI(0), th.NewMockMessagePoolValidator())
+		pool := NewMessagePool(th.NewTestMessagePoolAPI(0), th.NewTestMessagePoolConfig(maxMessagePoolSize, types.Uint64(10)), th.NewMockMessagePoolValidator())
```

Comment:
The test config construction generates a lot of noise. I recommend:
- move `NewMessagePoolConfig` next to the struct definition (drop `Test`), and do the `Uint64` conversion inside it.
- if you want, add `NewConfig()` as a private function here, with no parameters.